docs/src/main/sphinx/security/security-overview.rst

```diff
@@ -0,0 +1,159 @@
+=================
+Security overview
+=================
+
+After the initial :doc:`installation </installation>` of your cluster, security
+is the next major concern for successfully operating Trino. This overview
+provides an introduction to different aspects of configuring security for your
+Trino cluster.
+
+Aspects of configuring security
+-------------------------------
+
+The default installation of Trino has no security features enabled. Security
+can be enabled for different parts of the Trino architecture:
+
+* :ref:`security-client`
+* :ref:`security-inside-cluster`
+* :ref:`security-data-sources`
+
+Suggested configuration workflow
+--------------------------------
+
+To configure security for a new Trino cluster, follow this best practice
+order of steps. Do not skip or combine steps.
+
+1. **Enable** HTTPS
+
+* Work with your security team.
+* Use a load balancer or proxy to terminate HTTPS, if possible.
+* Use a globally trusted TLS certificate.
+
+2. **Enable authentication**
+
+* Start with a :doc:`password file <password-file>` to get up and running.
+* Then configure your preferred authentication provider, such as LDAP.
+* Avoid the complexity of Kerberos for client authentication, if possible.
+
+3. **Enable authorization and access control**
+
+* Start with :doc:`file-based rules <file-system-access-control>`.
+* Then configure another access control method as required.
+
+Configure one step at a time. Always restart the Trino server after each
+change and verify the results before proceeding.
+
+.. _security-client:
+
+Securing client access to the cluster
+-------------------------------------
+
+Trino *clients* include the Trino :doc:`CLI </installation/cli>`, the
+:doc:`Web UI </admin/web-interface>`, the :doc:`JDBC driver
+</installation/jdbc>`, and `community-provided clients, such as Python or Go
+clients <https://trino.io/resources.html>`_, and any applications using
+these tools.
```

Comment:
Maybe we should say that clients are typically used to write SQL queries and send them to Trino for processing or so? Or do we have a clients overview page we can link to?

Reply:
Linked.